libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/AndroidSpeechPlayer.java

```diff
@@ -33,8 +36,19 @@
     textToSpeech = new TextToSpeech(context, new TextToSpeech.OnInitListener() {
       @Override
       public void onInit(int status) {
-        if (status != TextToSpeech.ERROR) {
-          textToSpeech.setLanguage(new Locale(language));
+        if (status == TextToSpeech.ERROR || language == null) {
+          Timber.w("There was an error initializing native TTS");
```

Comment:
On second thought, what do you think about updating `Timber.w` to `Timber.e`?  If we are calling it an error in the log, I think it makes more sense.